lib/rspec-puppet-facts.rb

```diff
@@ -161,6 +178,13 @@ def self.spec_facts_os_filter
     ENV['SPEC_FACTS_OS']
   end
 
+  # If SPEC_FACTS_STRICT is set to `yes`, RspecPuppetFacts will error on missing FacterDB entries, instead of warning & skipping.
```

Comment:
This says 'skipping' but the behaviour is to attempt to use an older version of facts; so 'skipping' may be misleading